pkg/cvo/upgradeable.go

```diff
@@ -215,9 +216,28 @@ func (check *clusterVersionOverridesUpgradeable) Check() *configv1.ClusterOperat
 	return cond
 }
 
+type clusterDeletesInProgressUpgradeable struct {
```

Comment:
I think `manifestDeleteInProgressUpgradeable` or `clusterManifestDeleteInProgressUpgradeable` would be a better name for this type.

Reply:
I followed the pattern [here](https://github.com/openshift/cluster-version-operator/blob/2ea3d9e1fe2a0c2719c3b8fbda30610ce637df28/pkg/cvo/upgradeable.go#L239-L240). `cluster` is just meant to indicate that it is a cluster wide condition.

Reply:
Yeah. I saw that but did not know `cluster` was for clusterwide. I suggested `clusterManifestDeleteInProgressUpgradeable` because `ManifestDelete` is better to understand as compared to `Deletes`